.github/workflows/register-commands-staging.yaml

```diff
@@ -3,7 +3,31 @@ on:
   push:
     branches: develop
 jobs:
+  Environment-Variables-Check:
+    runs-on: ubuntu-latest
+    environment: staging
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.18.2
+      - run: npm install
+      - run: npm run env-var-check
+        env:
```

Comment:
NIT: Can we update checkout and node version to latest, I think 4 is latest or something
and same for node-version?

Reply:
sure, i will check and update it to latest version